developer/frameworks/articles/290-configuration/02-creating-a-configuration-interface.markdown

```diff
@@ -29,48 +30,44 @@ First, you'll learn how to create a configuration at the system scope.
-        value.  Note that if you set a field as required and don't specify a
-        default value, the system administrator must specify a value in order
-        for your application to work properly. Use the `deflt` property to
-        specify a default value.
+    **Meta.AD:** Specifies [optional
+    metadata](http://bnd.bndtools.org/chapters/210-metatype.html) about the
+    field, such as whether it's a required field or if it has a default
+    value.  Note that if you set a field as required and don't specify a
+    default value, the system administrator must specify a value in order
+    for your application to work properly. Use the `deflt` property to
+    specify a default value.
 
     The fully-qualified name of the `Meta` class above is
     `aQute.bnd.annotation.metatype.Meta`. For more information about this class and
     the `Meta.OCD` and `Meta.AD` annotations, please refer to the [bndtools
     documentation](http://bnd.bndtools.org/chapters/210-metatype.html).
 
 2.  To use the `Meta.OCD` and `Meta.AD` annotations in your modules, you must
-    [specify a dependency](/develop/tutorials/-/knowledge_base/7-0/configuring-dependencies)
+    [specify a dependency](/docs/7-2/customization/-/knowledge_base/c/configuring-dependencies)
     on the bnd library. We recommend using bnd version 3. Here's an example of
     how to include this dependency in a Gradle project: 
 
         dependencies {
             compile group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.1.0"
         }
 
-    +$$$
-
-    **Note:** The annotations `@Meta.OCD` and `@Meta.AD` are part of the bnd
-    library, but as of OSGi standard version R6, they're included in the OSGi core
-    under the names `@ObjectClassDefinition` and `@AttributeDefinition`. The OSGi
-    annotations can be used for simple cases like the one described in this
-    tutorial. However, a key difference between the two libraries is that the bnd
-    annotations are available at runtime, while the OSGi annotations are not.
-    Because runtime availability is necessary for some of the Liferay-specific features
-    described below, we recommend defaulting to the bnd annotations.
-
-    $$$
+    | **Note:** The annotations `@Meta.OCD` and `@Meta.AD` are part of the bnd
+    | library, but as of OSGi standard version R6, they're included in the OSGi core
+    | under the names `@ObjectClassDefinition` and `@AttributeDefinition`. The OSGi
+    | annotations can be used for simple cases like the one described in this
+    | tutorial. However, a key difference between the two libraries is that the bnd
+    | annotations are available at runtime, while the OSGi annotations are not.
+    | Because runtime availability is necessary for some of the Liferay-specific features
+    | described below, we recommend defaulting to the bnd annotations.
 
 3.  Add the following line to your project's `bnd.bnd` file:
```

Comment:
I've never had to add this line to my `bnd.bnd` file when adding configuration interfaces.  I think we add it automatically [here](https://github.com/liferay/liferay-portal/blob/ba6418fa48fae2c49feb4174a408a66e545d717d/build.properties#L26). @rotty3000 do we need to mention this as part of the the tutorial, or can we turn it into a side note?

Reply:
It could be a side note, but I'm sad that we use the bnd annotations. But yeah, let's turn it into a side note.

Reply:
@drewbrokke  can you confirm that this is what you had in mind?

https://raw.githubusercontent.com/nealrobbins/liferay-docs/LRDOCS-6630-configuration-interface/developer/frameworks/articles/290-configuration/02-creating-a-configuration-interface.markdown

Reply:
@nealrobbins that looks good to me, thanks!